src/cpu.py

```diff
@@ -67,9 +67,9 @@ def process(self, opcode, memory, io_device):
         """
         match opcode.name:
             case "READ":
-                self.read(memory, opcode.operand, iodevice)
+                self.read(memory, opcode.operand, io_device)  #changed iodevice to io_device. Scott
```

Comment:
Thanks for the comment but it's not necessary. =)

Reply:
Removed comments.